apps/website/src/components/footer/footer.tsx

```diff
@@ -0,0 +1,73 @@
+import { Text } from '@status-im/components'
+
+import { LINKS, SOCIALS } from '@/config/links'
+
+import { Logo } from '../logo'
+import { Dot } from './components/dot'
+import { MessariIcon } from './components/messari-icon'
+import { Section } from './section'
+
+type Props = {
+  hasBorderTop?: boolean
+}
+
+export const Footer = (props: Props) => {
+  const { hasBorderTop } = props
+
+  return (
+    <footer className="hidden pb-3 sm:block">
+      <div className="grid grid-cols-4 gap-5 sm:gap-0 lg:mb-10 lg:grid-cols-8">
+        <div
+          className={`border-neutral-80 mb-6 ${
```

Comment:
for conditional classes, please use `cx` helper from `class-variance-authority` package. It is easier to update and formats better than template literals.